src/poli/objective_repository/rosetta_energy/environment.yml

```diff
@@ -0,0 +1,12 @@
+name: poli__rosetta_energy
+channels:
+  - defaults
+  - conda-forge
+  - https://conda.rosettacommons.org
+dependencies:
+  - python=3.9
```

Comment:
Does rosetta have python 3.9 as a strong requirement? It would be nice if we could change this to python 3.10.